stack/stack_noaa.yaml

```diff
@@ -296,3 +296,28 @@ atlas:
 madis:
   build: YES
   version: 4.3
+
+esma_cmake:
+  build: YES
+  repo: GEOS-ESM
+  version: v3.4.3
+
+cmakemodules:
+  build: YES
+  repo: NOAA-EMC
+  version: develop
+
+gftl_shared:
+  build: YES
+  repo: Goddard-Fortran-Ecosystem
+  version: v1.3.0
+
+yafyaml:
+  build: YES
+  repo: Goddard-Fortran-Ecosystem
+  version: v0.5.1
+
+mapl:
+  build: YES
+  repo: GEOS-ESM
```

Comment:
are there multiple repos for this?
We should point to the authoritative repo.

Same applies for gftl_shared, and yafyaml.

Reply:
No. This is the authoritative repository: https://github.com/GEOS-ESM/MAPL